go.mod

```diff
@@ -121,7 +121,7 @@ require (
 
 replace (
 	// Our cosmos-sdk branch is:  https://github.com/osmosis-labs/cosmos-sdk  v0.44.3x-osmo-v5
```

Comment:
Make sure to update this commend

Reply:
That comment is correct still.

Reply:
We're using a branch instead of a release?